### PR TITLE
Streamline getting started JS docs

### PR DESCRIPTION
**Pull request by lizokm**

### CODE REVIEW

File: docs/platforms/javascript/common/index.mdx [hunk 1]
```diff
@@ -19,9 +17,16 @@ Check out the other SDKs we support in the left-hand dropdown.
 
 </Alert>
 
+## Prerequisites
+
+- Latest version of our [JavaScript SDK]((./migration/v7-to-v8/).
+- A [Sentry account]((https://sentry.io/signup/) and [Project](/product/projects/).
```
Comment (lizokm) [inline docs/platforms/javascript/common/index.mdx:23]:
```suggestion
- Latest version of our [JavaScript SDK](./migration/v7-to-v8/).
- A [Sentry account](https://sentry.io/signup/) and [Project](/product/projects/).
```

File: docs/platforms/javascript/common/index.mdx [hunk 2]
```diff
@@ -19,9 +17,16 @@ Check out the other SDKs we support in the left-hand dropdown.
...
+
+- Latest version of our [JavaScript SDK](./migration/v7-to-v8/).
+- A [Sentry account](https://sentry.io/signup/) and [Project](/product/projects/).
+
+<PlatformContent includePath="getting-started-primer" />
+
 </PlatformSection>
 
-Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+If you don't already have an account and a Sentry project set up, you can sign up for [Sentry](https://sentry.io/signup/) for free, then return to this page.
```
Comment (lizokm) [inline docs/platforms/javascript/common/index.mdx:29]:
```suggestion
Don't have a Sentry account? Sign up for [Sentry for free](https://sentry.io/signup/), then return to this page.
```

File: docs/platforms/javascript/common/index.mdx [hunk 3]
```diff
@@ -95,15 +100,9 @@ Sentry captures data by using an SDK within your application’s runtime.
 
 ## Configure
 
-<PlatformSection notSupported={["javascript.nextjs", "javascript.astro"]}>
-
-Configuration should happen as early as possible in your application's lifecycle.
-
-</PlatformSection>
-
```
Comment (lizokm) [inline docs/platforms/javascript/common/index.mdx:103]: <PlatformSection notSupported={["javascript.nextjs", "javascript.astro"]}>

Configuration should happen as early as possible in your application's lifecycle.

</PlatformSection>

File: docs/platforms/javascript/common/index.mdx [hunk 4]
```diff
@@ -95,15 +100,9 @@ Sentry captures data by using an SDK within your application’s runtime.
...
 <PlatformSection supported={["javascript.react"]}>
```
Comment (lizokm) [inline docs/platforms/javascript/common/index.mdx:103]:
```suggestion
<PlatformSection notSupported={["javascript.nextjs", "javascript.astro"]}>

Configuration should happen as early as possible in your application's lifecycle.

</PlatformSection>

<PlatformSection supported={["javascript.react"]}>
```

File: platform-includes/getting-started-config/javascript.react.mdx [hunk 5]
```diff
@@ -1,4 +1,4 @@
-It is important to initialize Sentry as early as possible in your application. We recommend putting the Sentry initialization code into its own file and including that file as the first import in your application entry point:
+Sentry should be initialized as early as possible in your application. We recommend putting the Sentry initialization code into its own file and including that file as the first import in your application entry point:
```
Comment (lizokm) [inline platform-includes/getting-started-config/javascript.react.mdx:1]:
```suggestion
We recommend putting the Sentry initialization code into its own file and including that file as the first import in your application entry point:
```

**Review by chargome — APPROVED** [review]

**Pull request merged**